.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
You'll want to add these otherwise it'll try to upgrade Lit on you:

```
ignore:
      # Lit 2 upgrade
      - dependency-name: "@open-wc/testing"
        versions: ["3.x"]
      # Lit 2 upgrade
      - dependency-name: "lit-element"
        versions: ["3.x"]
      # Lit 2 upgrade
      - dependency-name: "lit-html"
        versions: ["2.x"]
```